### PR TITLE
Remove deprecated code paths

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 6.0
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

I would need help to remove the deprecate code paths from the Doctrine bridge please :pray:

> 1) Symfony\Component\Security\Http\Tests\Firewall\AccessListenerTest::testHandleWhenThereIsNoAccessMapEntryMatchingTheRequest
> This test did not perform any assertions
> 
> /home/runner/work/symfony/symfony/src/Symfony/Component/Security/Http/Tests/Firewall/AccessListenerTest.php:77

Anyone has an idea why this failure appears?
It disappears if I revert the changes on `AbstractAuthenticatorTest`, but that doesn't make sense...

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
I would need help to remove the deprecate code paths from the Doctrine bridge please :pray:

**Comment by nicolas-grekas** [comment]
> 1) Symfony\Component\Security\Http\Tests\Firewall\AccessListenerTest::testHandleWhenThereIsNoAccessMapEntryMatchingTheRequest
> This test did not perform any assertions
> 
> /home/runner/work/symfony/symfony/src/Symfony/Component/Security/Http/Tests/Firewall/AccessListenerTest.php:77

Anyone has an idea why this failure appears?
It disapears if I revert the changes on `AbstractAuthenticatorTest`, but that doesn't make sense...

**Comment by stof** [comment]
> > 1. Symfony\Component\Security\Http\Tests\Firewall\AccessListenerTest::testHandleWhenThereIsNoAccessMapEntryMatchingTheRequest
> >    This test did not perform any assertions
> > 
> > /home/runner/work/symfony/symfony/src/Symfony/Component/Security/Http/Tests/Firewall/AccessListenerTest.php:77
> 
> Anyone has an idea why this failure appears?
> It disappears if I revert the changes on `AbstractAuthenticatorTest`, but that doesn't make sense...

I would say that this is related to what the test listener is doing:
https://github.com/symfony/symfony/blob/6bc0f354c0cc2a48ded008139ff4ff9e4e01d4d0/src/Symfony/Bridge/PhpUnit/Legacy/SymfonyTestsListenerTrait.php#L234-L246

It seems to detect only the presence of the trait, not the fact that it was actually called in the test.

**Comment by nicolas-grekas** [comment]
Does that also mean that `testHandleWhenThereIsNoAccessMapEntryMatchingTheRequest` is broken and that we didn't spot it because of that?

**Comment by stof** [comment]
Well, there is a mock assertion performed in it, but only conditionally:
https://github.com/symfony/symfony/blob/f1643e871dd8cb7612ddea6837de6acaaf49fca5/src/Symfony/Component/Security/Http/Tests/Firewall/AccessListenerTest.php#L168-L174

So the test is only doing no assertions in the 6.0 branch.

**Comment by nicolas-grekas** [comment]
Thanks for the help @stof, this should be good now.

File: src/Symfony/Component/Validator/Constraints/Range.php [hunk 1]
```diff
@@ -48,16 +48,6 @@ class Range extends Constraint
     public $max;
     public $maxPropertyPath;
 
-    /**
-     * @internal
-     */
-    public $deprecatedMinMessageSet = false;
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Constraints/Range.php:54]: Am I right removing this?

Comment (nicolas-grekas) [reply src/Symfony/Component/Validator/Constraints/Range.php:54]: Looks like I am.

**Comment by nicolas-grekas** [comment]
@HeahDude if you have some bandwidth, could you please have a look at removing the deprecated code from `src/Symfony/Bridge/Doctrine/Form/ChoiceList/DoctrineChoiceLoader.php` on branch 6.0?

**Pull request merged**